{
	"id": 66075881,
	"body": "\u003ca id=\"c9\"\u003e\u003c/a\u003eComment 9:\n\n\u003cpre\u003eI have caught the crash inside gdb:\n\n#0  runtime.memmove (to=void, fr=void, n=void) at\n/build/next/go/src/pkg/runtime/memmove_amd64.s:46\n#1  0x000000000041b5a2 in runtime.copy (to=void, fm=void, width=void, ret=void) at\n/build/next/go/src/pkg/runtime/slice.c:230\n#2  0x0000000000535e9c in bytes.(*Buffer).ReadFrom (fp=void, b=0xc20051bbd0, r=...,\nn=4294966338, err=0) at /build/next/go/src/pkg/bytes/buffer.go:162\n#3  0x00000000004778a4 in io.Copy (fp=void, dst=..., src=..., written=0, err=0) at\n/build/next/go/src/pkg/io/io.go:328\n#4  0x00000000005068a4 in os/exec.func·003 (fp=void, ~anon0=\"\u0026lt;bad dynamic type\u0026gt;\")\nat /build/next/go/src/pkg/os/exec/exec.go:207\n#5  0x0000000000506928 in os/exec.func·004 (fp=void, fn={void (error *)}\n0x7ff2040b2fb0) at /build/next/go/src/pkg/os/exec/exec.go:274\n#6  0x0000000000416760 in ?? () at /build/next/go/src/pkg/runtime/proc.c:1126\n#7  0x000000c2006e8e40 in ?? ()\n#8  0x0000000000000000 in ?? ()\n\nvoid stuff looks a bit suspect.\n\nn=4294966338 looks exciting...\u003c/pre\u003e",
	"user": {
		"login": "alberts",
		"id": 409689,
		"type": "User",
		"site_admin": false
	},
	"created_at": "2013-03-06T05:16:29Z",
	"updated_at": "2014-12-22T06:22:44Z"
}
